datanode/gateway/graphql/resolvers.go

```diff
@@ -2404,6 +2404,21 @@ func (r *myMarginLevelsUpdateResolver) MaintenanceLevel(_ context.Context, m *ve
 	return m.MaintenanceMargin, nil
 }
 
+func (r *myMarginLevelsUpdateResolver) OrderMarginLevel(_ context.Context, m *vegapb.MarginLevels) (string, error) {
+	return m.OrderMargin, nil
+}
+
+func (r *myMarginLevelsUpdateResolver) MarginFactor(_ context.Context, m *vegapb.MarginLevels) (string, error) {
+	return m.MarginFactor, nil
+}
+
+func (r *myMarginLevelsUpdateResolver) MarginMode(_ context.Context, m *vegapb.MarginLevels) (MarginMode, error) {
+	if m.MarginMode == vegapb.MarginMode_MARGIN_MODE_ISOLATED_MARGIN {
+		return MarginModeMarginModeIsolatedMargin, nil
```

Comment:
We usually map enums with in the gqlgen.yml file instead and add the marshaller in the marshallers package.